jans-cli-tui/cli_tui/plugins/999_jans/main.py

```
@@ -106,3 +128,139 @@ async def coroutine():
                 )
 
         asyncio.ensure_future(coroutine())
+
+
+    def get_iss_from_ssa(self, ssa):
+
+        try:
+            decoded = jwt.decode(ssa, options={"verify_signature": False, "verify_exp": True})
+        except jwt.DecodeError:
+            raise SSAError(_("Cant't decode SSA. Please check if it is valid jwt SSA"))
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)